src/pages/CommitDetailPage/CommitDetailPage.tsx

```diff
@@ -82,6 +82,7 @@ const CommitDetailPage: React.FC = () => {
         text: shortSHA,
       },
     ])
+    return () => setBreadcrumbs([])
```

Comment:
previously we were calling `setBreadcrumbs` with the branch when leaving these pages, but since we no longer are, we need to do this cleanup step on unmount.